doc/sphinx/source/inspection/viz_tools.rst

```diff
@@ -0,0 +1,42 @@
+.. inspection/viz_tools.rst:
+
+.. _viz_tools: 
+
+General Visualization Tools
+###########################
+
+nGraph provides serialization and deserialization facilities, along with the 
+ability to create image formats or a PDF. 
+
+When visualization is enabled, ``svg`` files for your graph get generated. The 
```

Comment:
`NGRAPH_ENABLE_VISUALIZE_TRACING=1` enables visualization. and generates graph visualization files.  
Note: this affects performance.
Can you add this note too?

Reply:
Done